vispy/gloo/buffer.py

```diff
@@ -162,7 +162,7 @@ def set_subdata(self, data, offset=0, copy=False, **kwargs):
         data : ndarray
             Data to be uploaded
         offset: int
-            Offset in buffer where to start copying data (in bytes)
+            Offset in buffer where to start copying data
```

Comment:
Could you maybe say that this is an index or is reference to the items or some other clear way of mentioning that this isn't bytes?

Reply:
Yes, indeed, it would be clearer. Not sure about how to best express it, though.
"(index into the underlying array)"?

Reply:
See tentative in the commit below.

Reply:
That works for me. I'm not sure how else to phrase this. At least we tried.